spec/spec_helper.rb

```diff
@@ -45,6 +91,22 @@
     mocks.verify_partial_doubles = true
   end
 
+  config.before(:suite) do
+    DatabaseCleaner.strategy = :transaction
+    DatabaseCleaner.clean_with(:truncation)
+    start_es_server unless ENV['ES_SKIP']
```

Comment:
Have you tried it? I just read through the docs of the vim-rspec plugin (https://github.com/thoughtbot/vim-rspec) and it seems to do the same thing I am doing manually.

My understanding is that `:suite` is shorthand for "all the individual tests that will run in this process" and so is invoked every time the `rspec` command executes.